portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,21 +44,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
+    //if user hasn't specified:
+    int userChoice = getChoice(request);
```

Comment:
for some reason userChoice is always 3

Reply:
This was resolved by adding the choice param to the fetch call